sa/sa.go

```diff
@@ -1375,11 +1378,50 @@ func (ssa *SQLStorageAuthority) NewAuthorizations2(ctx context.Context, req *sap
 		if err != nil {
 			return nil, err
 		}
-		err = ssa.dbMap.Insert(am)
+
+		// Each authz needs a (?,?...), in the VALUES block. We need one
+		// for each element in the authzFields string with no trailing ','.
+		fmt.Fprintf(&questionsBuf, "(%s),", strings.TrimRight(strings.Repeat(" ?,", 11), ","))
```

Comment:
nit: I think this as a constant string would __potentially__ look nicer and immediately standout to what it is referring to.

Reply:
I am good either way. The string would then be
```golang
		(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),
```
and we'd need to make sure it was 11.

Reply:
```suggestion
		fmt.Fprint(&questionsBuf, "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),")
```